packages/askui-nodejs/src/lib/interactive_cli/create-example-project.ts

```diff
@@ -23,28 +26,31 @@ export class CreateExampleProject {
   constructor(readonly cliOptions: CliOptions) {
     this.baseDirPath = process.cwd();
 
-    this.exampleFolderName = this.getExampleFolderName();
-    this.targetExampleFolderPath = path.join(
+    this.targetDirectoryName = this.getTargetFolderName();
+    this.targetDirectoryPath = path.join(
       this.baseDirPath,
-      this.exampleFolderName,
+      this.targetDirectoryName,
     );
 
     this.askUIControllerUrl = 'https://docs.askui.com/docs/general/Components/AskUI-Controller';
     this.helperTemplateConfig = {};
   }
 
-  private getExampleFolderName(): string {
-    if (this.cliOptions.projectName) {
-      return this.cliOptions.projectName.trim();
+  private getTargetFolderName(): string {
+    if (this.cliOptions.askuiWorkflows) {
```

Comment:
Don't like `askuiWorkflows` for several reasons

- testers or test automation engineers who are mostly using the lib at the moment, may not be familiar with this terminology, your other proposal `automations-dir` or `target-dir` seem better to me
- it is not clear that this refers to a directory at the moment
- we refer to this as `targetDirectory` throughout the code - why work with two terms if you can work with one (there may be good reasons to sometimes work with multiple terms for the same thing, in this case here I think it just adds complexity) either adjusting code or name of cli option.